source/winKernel.py

```diff
@@ -482,11 +482,11 @@ def SetThreadExecutionState(esFlags):
 
 
 def LCIDToLocaleName(windowsLCID: LCID) -> Optional[str]:
-	bufSize = 32
-	buf = ctypes.create_unicode_buffer(bufSize)
 	dwFlags = 0
-	kernel32.LCIDToLocaleName(windowsLCID, buf, bufSize, dwFlags)
+	bufferLength = kernel32.LCIDToLocaleName(windowsLCID, None, 0, dwFlags)

```

Comment:
If the function fails it will return 0 here and in the second case. Please check these return values.
https://docs.microsoft.com/en-us/windows/win32/api/winnls/nf-winnls-lcidtolocalename#return-value

Reply:
This was being implicitly checked by an empty buffer.
I've updated this to be more explicit, and add logging.